code/modules/mob/mob_helpers.dm

```diff
@@ -372,17 +372,17 @@
  */
 /proc/item_heal_robotic(mob/living/carbon/human/H, mob/user, brute_heal, burn_heal)
 	var/obj/item/bodypart/affecting = H.get_bodypart(check_zone(user.zone_selected))
-	if(affecting && affecting.status == BODYPART_ROBOTIC)
-		var/dam //changes repair text based on how much brute/burn was supplied
+	if(affecting && (!IS_ORGANIC_LIMB(affecting))) //PARIAH STATION EDIT
+	var/dam //changes repair text based on how much brute/burn was supplied
 		if(brute_heal > burn_heal)
 			dam = 1
 		else
 			dam = 0
 		if((brute_heal > 0 && affecting.brute_dam > 0) || (burn_heal > 0 && affecting.burn_dam > 0))
-			if(affecting.heal_damage(brute_heal, burn_heal, 0, BODYPART_ROBOTIC))
+			if(affecting.heal_damage(brute_heal, burn_heal, 0, BODYTYPE_ROBOTIC)) //PARIAH STATION EDIT
 				H.update_damage_overlays()
-			user.visible_message(span_notice("[user] fixes some of the [dam ? "dents on" : "burnt wires in"] [H]'s [affecting.name]."), \
-			span_notice("You fix some of the [dam ? "dents on" : "burnt wires in"] [H == user ? "your" : "[H]'s"] [affecting.name]."))
+			user.visible_message(span_notice("[user] has fixed some of the [dam ? "dents on" : "burnt wires in"] [H]'s [parse_zone(affecting.body_zone)]."), \ //PARIAH STATION EDIT
+			span_notice("You fix some of the [dam ? "dents on" : "burnt wires in"] [H == user ? "your" : "[H]'s"] [parse_zone(affecting.body_zone)].</span>")) //PARIAH STATION EDIT
```

Comment:
You don't need to put any `<span>` stuff in manually, since `span_notice()` does that for you:
https://github.com/pariahstation/Pariah-Station/blob/3cce7c6e98e2fa6be477a15186dca06c76bff893/code/__DEFINES/span.dm#L75
```suggestion
			user.visible_message(span_notice("[user] has fixed some of the [dam ? "dents on" : "burnt wires in"] [H]'s [parse_zone(affecting.body_zone)]."), \ //PARIAH STATION EDIT
			span_notice("You fix some of the [dam ? "dents on" : "burnt wires in"] [H == user ? "your" : "[H]'s"] [parse_zone(affecting.body_zone)].")) //PARIAH STATION EDIT
```